introduction.tex

```diff
@@ -20,22 +20,57 @@ \chapter{Introduction}
 
 \section{Terminology}
 
-A \emph{platform} is a single integrated circuit consisting of one or more
-\emph{components}. Some components may be RISC-V cores, while others may have a different
-function. Typically they will all be connected to a single system bus.
-A single RISC-V core contains one or more hardware threads, called
-\emph{harts}.
-
-\emph{DXLEN} of a hart is its widest supported XLEN, ignoring the current value
-of \Fmxl in \Rmisa.
-
-A \emph{physical address} is directly usable on the system bus.
-
-A \emph{virtual address} is an address as a hart sees it. If there is address
-translation this may be different from the \emph{physical address}. If there is
-no translation then it will be the same.
-
-\subsection{Context}
+\begin{description}[style=nextline]
+    \item[AMO]
+        Atomic Memory Operation.
+    \item[BYPASS]
+        JTAG instruction that selects a single bit data register, also called BYPASS.
+    \item[component]
+        A RISC-V core, or other part of a platform.
+        Typically all components will all be connected to a single system
+        bus.
+    \item[CSR]
+        Control and Status Register.
+    \item[DM]
+        Debug Module (see Section \ref{dm}).
+    \item[DMI]
+        Debug Module Interface (see Section \ref{dmi}).
+    \item[DR]
+        JTAG Data Register.
+    \item[DTM]
+        Debug Transport Module (see Section \ref{dtm}).
+    \item[DXLEN]
+        Debug XLEN, which is the widest XLEN a hart supports, ignoring the
+        current value of \Fmxl in \Rmisa.
+    \item[GPR]
+        General Purpose Register.
+    \item[hart]
+        A hardware thread in a RISC-V core.
+    \item[IDCODE]
+        32-bit Identification CODE, and a JTAG instruction that returns the IDCODE value.
+    \item[IR]
+        JTAG Instruction Register.
+    \item[JTAG]
+        Joint Test Action Group, referring to IEEE 1149.
+    \item[NAPOT]
+        Naturally Aligned Powers-Of-Two.
+    \item[NMI]
+        Non-Maskable Interrupt.
+    \item[hardware platform]
+        A single integrated circuit consisting of one or more \emph{components}.
```

Comment:
I didn't notice this the first time but it doesn't seem like it's necessarily a single integrated circuit.  Multi-chip modules, multi-socket systems, etc.  Fifteen years ago I worked on a multi-board system with seamless cache coherence between cores on the same chip, cores on different chips on the same board, and cores on chips on different boards.

Reply:
Sure. So what do I call it? A single system? A thing?

Reply:
I'd go with system.